app/stable_appconfig.go

```diff
@@ -72,7 +74,7 @@ func (app *RegenApp) registerUpgradeHandlers() {
 		panic(err)
 	}
 
-	if upgradeInfo.Name == "v0.43.0-beta1-upgrade" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+	if upgradeInfo.Name == upgradeName && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
```

Comment:
if we will have `v0.43.0-rc1`, we will need to update this again, right?
How about doing this change:
```
if stirngs.HasPrefix(upgradeInfo.Name, "v0.43.0") ...
```

Reply:
This is just the upgrade handler name, it may not refer to the original version.